data/governments.txt

```diff
@@ -788,6 +898,7 @@ government "Quarg"
 		"Hai (Wormhole Access)" .01
 		"Pirate" -.01
 		"Pirate (Devil-Run Gang)" -.01
+		"Gegno" 1
```

Comment:
That's the whole point - the Efret *should* do this. And this is necessary to protect them. They're lower tier, they are near a Quarg ringworld. I highly doubt the Quarg would watch you passively get shot at by Gegno ships without helping them if they deemed you a problem. Quarg ships should be attacking you on sight in Gegno systems they spawn, not wait for you to attack the hostile Gegno ships first.

Reply:
Getting a decent rep hit for attacking innocents is *more* in line with Quarg lore and philosophy than not, so their hostile response to you doing so is warranted.

Reply:
Perhaps this does need another Quarg government if that's truly an issue.

Reply:
This means we'd do the same for the Incipias then I assume since they're even less developed than the Gegno?

Reply:
Not sure, would have to ask Derpy/Leaf there.

Reply:
It might make sense to shift *both* the Gegno and the Efreti to 0.1, both for the arguments Saugia lists and the fact that the Efreti are literally and explicitly under Quarg protection.

Reply:
```suggestion
		"Gegno" 1
		"use foreign penalties for"
			"Gegno"
```
at the very least we need to add this or the Quarg will be pleased by you assisting Gegno even though they personaly dont care about it